notebooks/403-action-recognition-webcam/403-action-recognition-webcam.ipynb

```diff
@@ -0,0 +1,646 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# directory where model will be downloaded\n",
+    "base_model_dir = \"model\"\n",
+    "\n",
+    "# model name as named in Open Model Zoo\n",
+    "model_name = \"action-recognition-0001\"\n",
+    "# selected precision (FP32, FP16, FP16-INT8)\n",
+    "precision = \"FP16-INT8\"\n",
+    "\n",
+    "model_path = f\"model/intel/{model_name}/{model_name}/{precision}/{model_name}.xml\"\n",
+    "model_weights_path = f\"model/intel/{model_name}/{model_name}/{precision}/{model_name}.bin\"\n",
+    "\n",
+    "if not os.path.exists(model_path):\n",
+    "    download_command = f\"omz_downloader \" \\\n",
+    "                       f\"--name {model_name} \" \\\n",
+    "                       f\"--precision {precision} \" \\\n",
+    "                       f\"--output_dir {base_model_dir}\"\n",
+    "    ! $download_command\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Check the model path of your models\n",
+    "\n",
+    "We will be able to work with the models locally. Please check that you can see your models in your local machine after your download process."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Creating the proper path for architecture and weights model files.\n",
+    "model_path_decoder = f\"model/intel/{model_name}/{model_name}-decoder/{precision}/{model_name}-decoder.xml\"\n",
+    "model_weights_path_decoder = f\"model/intel/{model_name}/{model_name}-decoder/{precision}/{model_name}-decoder.bin\"\n",
+    "\n",
+    "model_path_encoder = f\"model/intel/{model_name}/{model_name}-encoder/{precision}/{model_name}-encoder.xml\"\n",
+    "model_weights_path_encoder = f\"model/intel/{model_name}/{model_name}-encoder/{precision}/{model_name}-encoder.bin\"\n",
+    "\n",
+    "# Testing the location of *.bin and *.xml\n",
+    "files = os.listdir(f\"model/intel/{model_name}/{model_name}-encoder/{precision}\")\n",
+    "print(files)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Load your labels\n",
+    "\n",
+    "We are using [Kinetics-400 dataset](https://deepmind.com/research/open-source/kinetics), and also we are providing you with the text file embedded into this notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# If you want to use another model, you should change the labels file\n",
+    "labels = \"data/kinetics.txt\"\n",
+    "\n",
+    "if labels:\n",
+    "    with open(labels) as f:\n",
+    "        labels = [line.strip() for line in f]\n",
+    "else:\n",
```

Comment:
This code was used just for a checking process on my side. If developers want to check another txt file with other labels, maybe it is simple, but it could help beginners understand the labels file and proper format. Do you think it is better to avoid the checking part?

Reply:
I like showing the labels! It is good to have a sanity check. I just would leave out the `if labels` part because it just checks that the string pointed to by `labels` is not empty.

Reply:
Done